src/figuresClasses.ts

```diff
@@ -2,10 +1,90 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side length must be greater than zero');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides cannot form a valid triangle');
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return area ? +area.toFixed(2) : 0;
+  }
+}
 
-export class Triangle implements Figure {}
+export class Circle implements Figure {
+  shape = 'circle';
 
-export class Circle implements Figure {}
+  color: string;
 
-export class Rectangle implements Figure {}
+  radius: number;
+
+  constructor(color: string, radius: number) {
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than zero');
+    }
+    this.color = color;
+    this.radius = radius;
+  }
+
+  getArea(): number {
+    const area = Math.PI * Math.pow(this.radius, 2);
+
+    return Math.floor(area * 100) / 100;
```

Comment:
The `getArea` method for the `Circle` class uses `Math.floor(area * 100) / 100` to truncate the area to two decimal places. Consider using `toFixed(2)` for consistency with the other classes and to ensure proper rounding.